software/delly/call/meta.yml

```diff
@@ -0,0 +1,55 @@
+name: delly_call
+description: Call structural variants
+keywords:
+  - genome
+  - structural
+  - variants
+  - bcf
+
+tools:
+  - delly:
+      description: Structural variant discovery by integrated paired-end and split-read analysis
+      homepage: https://github.com/dellytools/delly
+      documentation: https://github.com/dellytools/delly/blob/master/README.md
+      tool_dev_url: None
+      doi: "DOI:10.1093/bioinformatics/bts378"
+      licence: ["BSD-3-clause"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM file from alignment must be sorted, indexed, and duplicate marked
+      pattern: "*.{bam}"
+  - fasta:
+      type: file
+      description: The reference fasta file
+  - fastaidx:
+      type: file
+      description: Index of reference fasta file to identify split-reads
+  - exclude_regions:
+      type: file
+      description: |
+        A tsv file that excludes the positions of telomeres, centromeres, and unplaced contigs for a faster search (optional).
```

Comment:
This isn't an `input`?

Reply:
Woops! I forgot to take this out. I removed this in the end as it's an optional argument 👍 .